src/main/java/kusitms/backend/domain/user/entity/User.java

```diff
@@ -0,0 +1,34 @@
+package kusitms.backend.domain.user.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity(name="users")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+public class User {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private String provider;
```

Comment:
```suggestion
    @Column(nullable = false)
    @Enumerated(value = EnumType.STRING)
    private Provider provider;
```
enum 클래스 만들어 놨던데 그걸 사용해서 저장하는게 좋아요~